Plan/fabric/src/main/java/net/playeranalytics/plan/gathering/domain/FabricPlayerData.java

```diff
@@ -45,7 +45,7 @@ public UUID getUUID() {
 
     @Override
     public String getName() {
-        return player.getEntityName();
+        return player.getNameForScoreboard();
```

Comment:
Is there some other method for the player name? It sounds like an implementation detail that might be removed or renamed in the future

Reply:
Replaced with `getGameProfile().getName()`